ui/imports/shared/status/StatusChatInput.qml

```diff
@@ -601,51 +617,6 @@ Rectangle {
         return result
     }
 
-    function setFormatInInput(formationFunction, startTag, endTag, formationChar, numFormationChars) {
```

Comment:
Is this unused?

Reply:
Yes, removed in 249e6593b4f2571f5ba19dc4f6499d310b9248c6.